core/shared/src/main/scala/scala/macros/internal/Evidences.scala

```diff
@@ -0,0 +1,26 @@
+package scala.macros.internal
+
+trait Evidence1
```

Comment:
What are these for? Even being internal they still could use some documentation IMO